cmake/test.cmake

```diff
@@ -53,7 +53,7 @@ macro(get_gunit)
             NAME
             gunit
             GIT_TAG
-            467b07d
+            v1.16.0
```

Comment:
question: why use the version tag instead of the hash?

Reply:
If a version tag exists, we use it. This repo only uses hashes where a stable version that works for us doesn't exist.